app/main.py

```diff
@@ -1 +1,45 @@
-#  write your code here
+class Animal:
+    def __init__(self,
+                 name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        else:
```

Comment:
why do u need else here ?

Reply:
Бо у нас в умові задачі вказано. 

Animal should have two methods:

print_name - should print name in the following format: Hello, I'm {name}
feed - should print Eating {appetite} food points..., set is_hungry to False and return number of eaten food points if animal is hungry. _Otherwise, it should return 0 and print nothing._

Reply:
u could remove else condition and write return 0 on the same level as if condition 
`if <some condition>:`
`        return smth`
`return smth`
so this will work fine without else